### PR TITLE
Cta copy update

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Made copy changes on `/index` as per: https://docs.google.com/document/d/1b018GoKMF3abEFJlUkfAEK2JEhXWICKF4V0dGdTvgJs/edit#heading=h.8hni8w6zn8zi

- Added CTA on `/compare` as per: https://docs.google.com/document/d/1JngYmRhohPo62yU72V1xpzCCILI21SmHKKttBxHjPog/edit#

## QA

- View the site locally in your web browser at: https://ubuntu-com-10443.demos.haus/kubernetes and https://ubuntu-com-10443.demos.haus/kubernetes/compare
- Check relevant pages against copydocs and see that all requested changes have been made

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10434

### CODE REVIEW

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -283,42 +284,55 @@ <h2 class="p-muted-heading u-align--center">FULLY-SUPPORTED KUBERNETES PRIVATE C
           }}
         </li>
         <li class="p-inline-images__item">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/760109ef-CloudSigma_Logo_Full.png",
-            alt="Cloud Sigma",
-            width="130",
+          {{ image (
```
Comment (minkyngkm) [inline templates/kubernetes/index.html:287]: I think [the inline images pattern](https://vanillaframework.io/docs/patterns/inline-images) is deprecated so we need to use [the logo section pattern ](https://vanillaframework.io/docs/patterns/logo-section)instead.

**Review by minkyngkm — CHANGES_REQUESTED** [review]
Looks good MariaPaula! 👍 Just need to update the inline images pattern I think!

**Comment by minkyngkm** [comment]
Oh also in the `/kubernetes/compare`page this part seems not to be updated yet according to the copy doc. 
image

**Comment by minkyngkm** [comment]
> > copy doc
> 
> I think that's actually correct, the title for `/index`, `/what-is-kubernetes`, and `/compare` should be "Talk to our Kubernetes experts". If you scroll all the way up on that copydoc you'll see it :)

Ohhh yes you are right I thought "Let us manage your Kubernetes" was default! Sorry! 😝

**Comment by mtruj013** [comment]
No worries! I made the same mistake when I first saw it too. I'm changing the logos now, there's another section of the page that has those deprecated classes as well so I'm fixing those and I'll let you know when it's ready :)

File: templates/kubernetes/index.html [hunk 2]
```diff
@@ -204,9 +204,10 @@ <h2>Kubernetes with Canonical support</h2>
           </li>
           <li class="p-list__item is-ticked">Fine-grained service placement</li>
         </ul>
-        <p>
-          <a href="/kubernetes/install#multi-node">Install Charmed Kubernetes&nbsp;&rsaquo;</a>
-        </p>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/kubernetes/install#multi-node">Install Charmed Kubernetes&nbsp;&rsaquo;</a></li>
+          <li class="p-inline-list__item"><a class="p-link--external" href="https://assets.ubuntu.com/v1/c5f59e1c-Enterprise_Kubernetes+Datasheet_16.09.21.pdf">Read the datasheet</a></li>
```
Comment (minkyngkm) [inline templates/kubernetes/index.html:209]: I just got to know that it doesn't require an external link if it is for something on pages.ubuntu.com or assets.ubuntu.com, like ebooks, whitepapers, pdfs, etc. so it doesn't seem to be required the `p-link--external` class here.

```suggestion
          <li class="p-inline-list__item"><a href="https://assets.ubuntu.com/v1/c5f59e1c-Enterprise_Kubernetes+Datasheet_16.09.21.pdf">Read the datasheet&nbsp;&rsaquo;</a></li>
```

File: templates/kubernetes/index.html [hunk 3]
```diff
@@ -204,9 +204,10 @@ <h2>Kubernetes with Canonical support</h2>
           </li>
           <li class="p-list__item is-ticked">Fine-grained service placement</li>
         </ul>
-        <p>
-          <a href="/kubernetes/install#multi-node">Install Charmed Kubernetes&nbsp;&rsaquo;</a>
-        </p>
+        <ul class="p-inline-list">
```
Comment (minkyngkm) [inline templates/kubernetes/index.html:207]: I think its just `p-list`
```suggestion
        <ul class="p-list">
```

**Review by minkyngkm — APPROVED** [review]
Looking Good MariaPaula!👍 Just a comment regarding the external link! :)

File: templates/kubernetes/index.html (hunk 3 above)
Comment (mtruj013) [reply templates/kubernetes/index.html:207]: It's `p-list` for regular lists but `p-inline-list` for [inline lists](https://vanillaframework.io/docs/patterns/lists#inline) 🙂

**Pull request merged**